.github/workflows/build.yaml

```diff
@@ -0,0 +1,167 @@
+name: Build binaries
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    name: Build binary packages
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Check https://github.com/livepeer/go-livepeer/pull/1891
+          # for ref value discussion
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: "yarn"
+
+      - name: Tags
+        id: tags
+        uses: livepeer/action-gh-release-tags@latest
+
+      - name: restore lerna
+        id: cache-lerna
+        uses: actions/cache@v3
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: restore pkg cache
```

Comment:
Same regarding cache 🤔

Reply:
In case of sub-packages (like what's happening in `api.yaml`), it does work as expected. With lerna; we have separate `node_modules` populated for each package (api/www/admin/design-system) and the restore doesn't work as expected. The cache from `setup-node` only restores the root `node_modules` dir (check cache sizes). Although, the yarn step still takes 10m despite having the packages available so I blame nodejs :smile: 

The caching of `pkg` is entirely for the built binaries. It has its own caching independent from yarn/npm. Check `PKG_CACHE_PATH` at <https://github.com/vercel/pkg#environment>.